### PR TITLE
fixes 133

### LINKED ISSUE #133: HTTP 500 when no entries exist

### PR DESCRIPTION
**Pull request by sfarmar**
Fixes #133

### CODE REVIEW

File: src/HashBus.WebApi/MostHashtaggedModule.cs [hunk 1]
```diff
@@ -35,8 +35,9 @@ public MostHashtaggedModule(IRepository<string, IEnumerable<Hashtag>> hashtags)
                 {
                     Entries = entries,
                     Count = trackHashtags.Count,
-                    Since = trackHashtags.Min(hashtag => hashtag.HashtaggedAt),
-                    LastActivityDateTime = trackHashtags.Max(hashtag => hashtag.HashtaggedAt),
+
```
Comment (adamralph) [inline src/HashBus.WebApi/MostHashtaggedModule.cs]: Sorry for the nitpick, but can we remove this added line?

File: src/HashBus.WebApi/MostHashtaggedModule.cs [hunk 2]
```diff
@@ -35,8 +35,9 @@ public MostHashtaggedModule(IRepository<string, IEnumerable<Hashtag>> hashtags)
                 {
                     Entries = entries,
                     Count = trackHashtags.Count,
-                    Since = trackHashtags.Min(hashtag => hashtag.HashtaggedAt),
-                    LastActivityDateTime = trackHashtags.Max(hashtag => hashtag.HashtaggedAt),
+
+                    Since = trackHashtags.Any() ? trackHashtags.Min(hashtag => hashtag.HashtaggedAt) : (DateTime?)null,
+                    LastActivityDateTime = trackHashtags.Any() ? trackHashtags.Max(hashtag => hashtag.HashtaggedAt) : (DateTime?)null
```
Comment (adamralph) [inline src/HashBus.WebApi/MostHashtaggedModule.cs]: Also another nitpick, but this line has the trailing comma removed. It's a style thing, but personally I prefer to have every line in an initialiser to have a trailing comma so they are consistent, can be moved around, don't have to be altered when a new line is added to the end of the list, etc.

**Comment by adamralph** [comment]
Thanks @sfarmar!

It looks good, I just left a couple of very minor nitpick comments. (The trailing comma comment applies to all 6 modules.)

File: src/HashBus.WebApi/MostHashtaggedModule.cs (hunk 1 above)
Comment (sfarmar) [inline src/HashBus.WebApi/MostHashtaggedModule.cs]: Yup, cleaned up

File: src/HashBus.WebApi/MostHashtaggedModule.cs (hunk 2 above)
Comment (sfarmar) [inline src/HashBus.WebApi/MostHashtaggedModule.cs]: yup, I get it, updated all

**Comment by adamralph** [comment]
Thanks @sfarmar, LGTM :+1: 

Would you mind squashing the two commits and force pushing? Then I'll be happy to merge.

**Comment by sfarmar** [comment]
Squashed

**Comment by adamralph** [comment]
:ok_hand: perfect @sfarmar, thanks!

**Pull request merged**